source/documentation/data_warehouse_guide/topics/Virtual_machine_disk_hourly_and_samples_history_views.adoc

```diff
@@ -30,15 +30,15 @@
 |max_read_rate_bytes_per_second |integer |The maximum read rate for the aggregation period. For hourly aggregations, this is the maximum collected sample value. For daily aggregations, it is the maximum hourly average value. |No
 |read_ops_per_second |bigint |The number of read I/O operations per second to disk.
```

Comment:
The column read_ops_per_second is read_ops_total_count, so remove this line.

Reply:
removed redundant read_ops_per_second